dags/pytorch_xla/configs/pytorchxla_torchbench_config.py

```diff
@@ -221,7 +223,7 @@ def get_torchbench_gpu_config(
 
   set_up_cmds = set_up_torchbench_gpu(model_name, nvidia_driver_version)
   local_output_location = "/tmp/xla/benchmarks/output/metric_report.jsonl"
-  gcs_location = f"{gcs_bucket.BENCHMARK_OUTPUT_DIR}/torchbench_config/{accelerator_type}/{int(datetime.datetime.now().timestamp())}/metric_report_gpu.jsonl"
+  gcs_location = f"{gcs_bucket.BENCHMARK_OUTPUT_DIR}/torchbench_config/{accelerator_type}/metric_report_gpu.jsonl"
```

Comment:
I think `{int(datetime.datetime.now().timestamp())}` can be fixed by a xcom or variable in Airflow. It may not be a good idea to delete your benchmarking report for previous runs.

Reply:
I can change to use xcom. My only concern is that this will add an extra function entry in `task.py` like `run_with_gcs_name`. Which one do you prefer?